Phonebook.Frontend/src/i18n/messages.de.xlf

```diff
@@ -402,6 +402,14 @@
             </target>
         <note priority="1" from="description">The alphabetical desc order of the favorite cards</note>
         <note priority="1" from="meaning">DashboardComponent</note>
+      </trans-unit><trans-unit id="DashboardComponentButtonExpandTooltip" datatype="html">
+        <source>Expand or Collapse recently viewed people</source><target state="translated">Diesen Bereich ein- oder ausklappen.</target>
+        <note priority="1" from="description">Tooltip for expanding recently viewed people</note>
+        <note priority="1" from="meaning">DashboardComponent</note>
+      </trans-unit><trans-unit id="DashboardComponentButtonExpandArialLabel" datatype="html">
+        <source>Button expanding or collapsing the recently viewed people section</source><target state="new">Knop um den Bereich kürzlich gesuchte Personen zu schließen oder zu öffnen</target>
```

Comment:
Wir sind zwar in Sachsen aber wenn dann bitte Knobb! Babsches B bitte!
Spaß beiseite. Mach da mal das draus:

```suggestion
        <source>Button expanding or collapsing the recently viewed people section</source><target state="new">Knopf um den Bereich kürzlich gesuchte Personen zu schließen oder zu öffnen</target>
```